lms/djangoapps/edraak_university/admin.py

```diff
@@ -0,0 +1,59 @@
+"""
+Django admin for university ID.
+"""
+from django.contrib import admin
+from django.core.urlresolvers import reverse
+
+from student.models import UserProfile
+from openedx.core.djangolib.markup import HTML, Text
+
+from edraak_university.models import UniversityID
+
+
+@admin.register(UniversityID)
+class UniversityIDAdmin(admin.ModelAdmin):
+    """Admin for University Student ID."""
+    list_display = (
+        'edraak_user',
+        'course_key',
+        'university_id',
+        'section_number',
+        'edraak_profile',
+        'email',
+        'date_created',
+    )
+
+    readonly_fields = ('user', 'course_key', 'date_created',)
+    search_fields = ('user__email', 'user__username', 'course_key', 'university_id',)
+
+    def edraak_user(self, university_id_obj):
+        """
+        Provides the username with a link for the User object in the admin.
+        """
+        user = university_id_obj.user
+        return HTML(u'<a href="{url}">{name}</a>').format(
+            name=Text(unicode(user)),
+            url=Text(reverse('admin:auth_user_change', args=[user.pk])),
+        )
+
+    def edraak_profile(self, university_id_obj):
+        """
+        Provides a link to the UserProfile object in the admin.
+        """
+        try:
+            edraak_profile = UserProfile.objects.get(user=university_id_obj.user)
+        except UserProfile.DoesNotExist:
+            return None
+
+        return HTML(u'<a href="{url}">Profile</a>').format(
+            url=Text(reverse('admin:student_userprofile_change', args=[edraak_profile.pk])),
```

Comment:
This will return NoReverseMatch, or 500 in production, please check ```student/admin.py``` for more details on new changes.